Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -294,7 +294,8 @@ public partial class BlockPolicySource
                     transaction.Actions.First().InnerAction is IActivateAction aa)
                 {
                     return blockChain.GetState(aa.GetPendingAddress()) is Dictionary rawPending &&
-                        new PendingActivationState(rawPending).Verify(aa.GetSignature())
+                        new PendingActivationState(rawPending).Verify(aa.GetSignature()) &&
+                        transaction.Nonce == 0
```

Comment:
Tiny optimization but we can skip unnecessary `GetState()` & `PendingActivationState.Verify()` calls if we move nonce checking to the first.

Reply:
Fixed.